app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -95,24 +96,24 @@ public void signUp() {
         if(personalServer.isChecked()) {
             if(!CredentialHelper.checkIfEmpty(url,this) && CredentialHelper.isURLValid(url,this)) {
                 if (CredentialHelper.getValidURL(url,this) != null) {
-                    PrefManager.putBoolean("is_susi_server_selected", false);
-                    PrefManager.putString("custom_server", CredentialHelper.getValidURL(url, this));
+                    PrefManager.putBoolean(Constant.SUSI_SERVER, false);
+                    PrefManager.putString(Constant.CUSTOM_SERVER, CredentialHelper.getValidURL(url, this));
                 } else {
-                    url.setError("Invalid URL");
+                    url.setError(Constant.INVALID_URL);
```

Comment:
@madhavrathi don't add every string in the constant class. Only those strings which are used to store pref should be added to the constant because we don't want translations of them. The concept of adding strings to string.xml file is that if the user changes his language from settings, the app should be able to show every string in that language. This is done by adding strings.xml file for that language and doing translations of all the string. If you add them in constant class, they will not be translated into other languages.